src/robot/running/suiterunner.py

```diff
@@ -125,6 +125,11 @@ def visit_test(self, test):
         if status.exit:
             self._add_exit_combine()
             result.tags.add('robot:exit')
+        if TagPatterns("robot:skip").match(test.tags):
+            status.test_skipped(
+                test_or_task(
+                    "{Test} skipped since test is tagged with robot:skip.",
+                    self._settings.rpa))
```

Comment:
Cannot use word `test` as-is in the message because we want to use `task` in RPA mode. That `test_or_task` helper converts placeholders like `{Test}` to right format automatically, but I wouldn't repeat test/task in one sentence anyway. Perhaps something like `{Test} skipped because it had 'robot:skip' tag.` would be fine.

Reply:
Above comment already handled. Two nitpicky things to still change:
- Add `.` at the end of the sentence.
- You can use `"` around the string to be able to use `'` in the string without escaping like `"{Test} ... 'robot:skip' tag."`